dongken/app.js

```diff
@@ -0,0 +1,108 @@
+require('dotenv').config();
+
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+
+
+const {DataSource} = require('typeorm');
+
+const appDateSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+appDateSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+app = express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('combined'));
+
+app.get("/ping", (req,res)=> {
+    res.status(201).json({message : "pong"})
+});
+
+app.post('/users/signup', async (req, res) => {
+	const { name, email, profile_image, password} = req.body
+
+    
+    await appDateSource.query(
+		`INSERT INTO users(
+		    name,
+		    email,
+		    profile_image,
+            password
+		) VALUES (?, ?, ?, ?);
+		`,
+		[ name, email, profile_image, password]
+	); 
+     res.status(201).json({ message : "userCreated" });
+	})
+
+ app.post('/posts', async (req, res) => {
+	const { title, content, image_url, user_id} = req.body
+    
+	await appDateSource.query(
+		`INSERT INTO posts(
+		    title,
+		    content,
+		    image_url,
+            user_id
```

Comment:
동근 SQL 쿼리문 포맷팅을 지켜서 작성해 주세요 !
```suggestion
	await appDateSource.query(
		`INSERT INTO posts(
		    title,
		    content,
		    image_url,
                    user_id
```